### DISCUSSION TITLE
Copilot is not working in my WebStorm IDE without showing any errors.

### DISCUSSION THREAD

**Discussion by andynazl** [community/community#125144] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I have followed the instructions in the [Getting Started with GitHub Copilot](https://docs.github.com/en/copilot/using-github-copilot/getting-started-with-github-copilot) guide to download and configure the GitHub Copilot plugin in WebStorm, but it isn't working properly. There is no auto-completion, and Copilot Chat is also unavailable. However, it works normally in Visual Studio Code.
And then i tried clicking on 'Log Diagnostics,' but WebStorm immediately froze, forcing me to quit.
I also attempted to configure '#com.github.copilot:trace' to collect logs, but didn't find any useful information.

**Comment by andynazl** [comment, marked as answer]
The problem has been identified; it was caused by the proxy settings that resulted in connection issues with GitHub Copilot. The issue was resolved after modifying the proxy configuration.

**Reply by Srabon444** [reply]
Can you let me know the details how you have modified the proxy?

**Reply by andynazl** [reply]
> Can you let me know the details how you have modified the proxy?
The key point is that the domain name used by Copilot has been blocked by me.
eg: `- DOMAIN-SUFFIX,githubcopilot.com,proxy`
